gabbi/handlers/core.py

```diff
@@ -22,8 +22,20 @@ class StringResponseHandler(base.ResponseHandler):
     test_key_value = []
 
     def action(self, test, expected, value=None):
-        expected = test.replace_template(expected)
-        test.assert_in_or_print_output(expected, test.output)
+        is_regex = (expected.startswith('/') and
+                    expected.endswith('/') and
+                    len(expected) > 1)
```

Comment:
I believe this is now the third occurrence of that particular detection logic, so how about extracting it into a function?

```python
def is_regex(str):
    return re.match("^/.+/$", str) is not None
```

It might additionally do or prepare the `assertRegex` dance beneath.

Reply:
Yeah, good idea, but  I'll stick with starts and ends with.

Which is perhaps ironic, but there you go.

I'll avoid preparing the dance to keep concerns separated.